zipkin-lens/src/util/global-search.js

```diff
@@ -12,6 +12,7 @@ export const lookbackDurations = {
 
 export const orderedConditionKeyList = autocompleteKeys => ([
   'serviceName',
+  'remoteServiceName',
```

Comment:
`orderedConditionKeyList` determines the order of condition badges.
In other words, this defines the priority of the condition display.
Do users frequently search traces using `remoteServiceName` as search criteria?
If not, I think this search condition should be placed a bit further down.

Reply:
Many users are used to `select serviceName → select spanName` flow.
So if the `remoteServiceName` is in the current position, they may be a little confused...

Reply:
I guess we don't know how frequently this will be used. I can move it down and people can ask for it to be moved up if they want :)

Reply:
In my opinion, to avoid confusion `remoteServiceName` should be after `spanName` or after `maxDuration` at first.

Reply:
roger

Reply:
actually I put it last as not sure how often people will use this